src/index.js

```diff
@@ -14,6 +14,8 @@ export default class ExampleComponent extends React.PureComponent {
     loaderSize: PropTypes.string,
     color: PropTypes.string,
     textStyles: PropTypes.object,
+    wrapperStyles: PropTypes.object,
```

Comment:
Could you add these 2 props to the readme file and a short explanation? Thanks

Reply:
@felipe1234-dev

Reply:
Really sorry for taking so long to reply. I was very busy with my project (that uses yours) and I just forgot about everything else. So, yep, I'm gonna do that.